src/components/common/footer/footer.scss

```diff
@@ -1,3 +1,22 @@
+.links-container {
+  display: flex;
+  justify-content: center;
+
+  a {
+    padding: 5px;
```

Comment:
Add some padding in between the icons to give more room for thumbs to click.

Also, something is going on with the sizing. If it's not too annoying, try to fix it. If the FA library makes it too hard, ignore the sizing thing.

<img width="206" alt="Screen Shot 2020-07-24 at 12 25 10 PM" src="https://user-images.githubusercontent.com/14047327/88419121-32e19380-cdaa-11ea-8a8f-744b831003c2.png">